### ISSUE TITLE
Make sure tombstone button only shows up for published works. 

### ISSUE THREAD

**Issue by straleyb** [osulp/Scholars-Archive#2066]
### Descriptive summary

Works that haven't been published, should not be tombstoned. Dont show the tombstone button unless the work is actually deposited.

**Comment by simholt** [comment]
So it looks like items that were already _in review_ are good -- tombstone buttons did not appear until the items were approved.

I deposited items in every worktype and most worked properly. Some had issues.

**Skipped queue + no tombstone button when approved:**
- Honors College Thesis (2-part approval process, like ETDs)
- Purchased E-Resources (single approval)

**No tombstone button:**
- Article (does not go through review process)

**Tombstone button appeared after review, all good:**
- ETD
- Undergrad Thesis
- Graduate Project
- Default
- Technical Report
- Dataset
- EESC Publication
- OER
- Conference Proceeding
- Admin Report (tombstone button appeared after approval but the upload was set to only Private visibility, could not change. Will create separate ticket.)

**Comment by luisgreg99** [comment]
I was unable to recreate the issue for Honors College Thesis, Purchased E-Resources, and Article in my local tests. 

After reviewing the related PRs, the issue appears to have been fixed by https://github.com/osulp/Scholars-Archive/pull/2074 which enables the tombstone action in `default` and `Self Deposit` workflows. I have reloaded all the workflows on deploy, so this fix is now be available on staging.
